packages/sdk-cli/src/commands/screenshot.test.ts

```diff
@@ -71,6 +76,12 @@ it('logs the location of the file to the console', async () => {
   expect(mockUIRedraw.done).toBeCalled();
 });
 
+it('opens the screenshot', async () => {
+  const destPath = path.resolve('Screenshot 2018-03-04 at 15.06.07.png');
+  await expectDestPath(destPath, undefined, true);
```

Comment:
nit: Maybe rename `expectDestPath` to something like `expectCaptureScreenshotCall`?

Reply:
Thanks, I renamed it